integration_tests/e2e/offenceEditOffence.cy.ts

```diff
@@ -185,16 +185,28 @@ context('Add Offence Edit offence Page', () => {
       })
       offenceEditOffencePage.editFieldLink('A1234AB', 'add', '0', '0', '0', 'sentence-type').click()
       const offenceSentenceTypePage = Page.verifyOnPage(OffenceSentenceTypePage)
-      offenceSentenceTypePage.radioSelector('467e2fa8-fce1-41a4-8110-b378c727eed3').should('be.checked')
+      offenceSentenceTypePage.radioSelector('467e2fa8-fce1-41a4-8110-b378c727eed3|STANDARD').should('be.checked')
       offenceSentenceTypePage.radioLabelContains('EDS (Extended Determinate Sentence)').click()
       offenceSentenceTypePage.button().click()
+      let offencePeriodLengthPage = Page.verifyOnPageTitle(OffencePeriodLengthPage, 'overall sentence length')
```

Comment:
config defined https://github.com/ministryofjustice/hmpps-remand-and-sentencing/blob/main/server/resources/sentenceTypePeriodLengths.ts#L9-L20 because the sentence type has changed from standard to extended the user must now enter all the period lengths for an extended sentence before going back to the edit page